src/app/content/externalClickHandler.ts

```diff
@@ -2,41 +2,16 @@ import { MouseEvent } from 'react';
 import { Store } from 'redux';
 import { close } from 'libs/store/actions/ui';
 import { CloseCause } from 'libs/domain/ui';
+import { isEventPathInteractive } from '../../libs/utils/isEventPathInteractive';
 import { isOpen } from './store/selectors';
 
-export const interactiveElementsSelectors: string[] = [
-  'a',
-  'details',
-  'dialog',
-  'menu',
-  'menuitem',
-  'summary',
-  'button',
-  'datalist',
-  'input',
-  'label',
-  'meter',
-  'optgroup',
-  'option',
-  'select',
-  'textarea',
-  '[onclick]'
-];
-
-export const isHtmlElementInteractive = (element?: HTMLElement | null) =>
-  element &&
-  element.matches &&
-  interactiveElementsSelectors.some(selector => element.matches(selector));
-
-export default (store: Store) => (e: MouseEvent) => {
+interface ExtendMouseEvent extends MouseEvent {
+  composedPath: () => HTMLElement[];
+}
+export default (store: Store) => (e: ExtendMouseEvent) => {
   const state = store.getState();
-
-  const element = e.target as HTMLElement;
-  const parentElement = element.parentElement;
-
-  const interactive =
-    isHtmlElementInteractive(element) ||
-    isHtmlElementInteractive(parentElement);
+  const pathElements = e.composedPath && e.composedPath();
+  const interactive = isEventPathInteractive(pathElements);
```

Comment:
You should cast `pathElements` as `HtmlElement` instead of overriding the `composedPath` function definition. This is one of the few valid use case for casting:
```ts
const interactive = isEventPathInteractive(pathElements as HTMLElement[]);
```
> https://developer.mozilla.org/en-US/docs/Web/API/Event/composedPath